tests/onnx/test_onnx_metatypes.py

```diff
@@ -0,0 +1,38 @@
+from collections import Counter
+
+import pytest
+from nncf.experimental.onnx.graph.nncf_graph_builder import GraphConverter
+
+from nncf.common.graph.operator_metatypes import InputNoopMetatype
+from nncf.common.graph.operator_metatypes import OutputNoopMetatype
+from nncf.common.graph.operator_metatypes import UnknownMetatype
+from nncf.experimental.onnx.graph.metatypes.onnx_ops import ConvolutionMetatype
+from nncf.experimental.onnx.graph.metatypes.onnx_ops import BatchNormMetatype
+from nncf.experimental.onnx.graph.metatypes.onnx_ops import ReluMetatype
+from nncf.experimental.onnx.graph.metatypes.onnx_ops import GlobalAveragePoolMetatype
+from nncf.experimental.onnx.graph.metatypes.onnx_ops import ConcatLayerMetatype
+from nncf.experimental.onnx.graph.metatypes.onnx_ops import AddLayerMetatype
+
+from tests.onnx.models import LinearModel
+from tests.onnx.models import MultiInputOutputModel
+from tests.onnx.models import ModelWithIntEdges
+
+TEST_MODELS = [LinearModel, MultiInputOutputModel, ModelWithIntEdges]
+REF_METATYPES_COUNTERS = [
+    [InputNoopMetatype, ConvolutionMetatype, BatchNormMetatype, ReluMetatype, GlobalAveragePoolMetatype,
```

Comment:
How does it work now with metatype, if you only plan to support metatypes in next PR (ticket 58797)?